### ISSUE TITLE
Overly strict get input in ElectricEquipmentITEAirCooled

### ISSUE THREAD

**Issue by jmarrec** [NREL/EnergyPlus#7117]
Issue overview
--------------

The object has a choice field `Air Inlet Connection Type`.

Field A15 and A16 are two curves that per IDD are only used if the above connection type is `AdjustedSupply`, and if omitted should default to always 1.0, but if you see your object to AdjustedSupply and omit these two, E+ Fatals out.

See portion of code in [InternalHeatGains.cc#L3778:L3798](https://github.com/NREL/EnergyPlus/blob/b0d6191edfecd4a35971336365ef893651e55199/src/EnergyPlus/InternalHeatGains.cc#L3778:L3798)

**IDD extract**:

```
  A11, \field Air Inlet Connection Type
       \note Specifies the type of connection between the zone and the ITE air inlet node.
       \note AdjustedSupply = ITE inlet temperature will be the current Supply Air Node temperature
       \note adjusted by the current recirculation fraction.
       \note All heat output is added to the zone air heat balance as a convective gain.
       \note ZoneAirNode = ITE air inlet condition is  the average zone condition.
       \note All heat output is added to the zone air heat balance as a convective gain.
       \note RoomAirModel = ITE air inlet and outlet are connected to room air model nodes.
       \note This field is only used when Air Flow Calculation Method is FlowFromSystem.
       \type choice
       \key AdjustedSupply
       \key ZoneAirNode
       \key RoomAirModel
       \default AdjustedSupply
[...]
  A15, \field Recirculation Function of Loading and Supply Temperature Curve Name
       \type object-list
       \object-list BivariateFunctions
       \note The name of a two-variable curve or table lookup object which modifies the recirculation
       \note fractionas a function of CPU loading (x) and supply air node temperature (y).
       \note This curve (table) should equal 1.0 at design conditions (CPU loading = 1.0 and
       \note Design Entering Air Temperature).This field is used only if the
       \note Air Node Connection Type = AdjustedSupply. If this curve is left blank, then the curve
       \note is assumed to always equal 1.0.
       \note This field is only used when Air Flow Calculation Method is FlowFromSystem.
[...]
  A16, \field Electric Power Supply Efficiency Function of Part Load Ratio Curve Name
       \type object-list
       \object-list UnivariateFunctions
       \note The name of a single-variable curve or table lookup object which modifies the electric
       \note power supply efficiency as a function of part-load ratio (x).
       \note This curve (table) should equal 1.0 at full load (PLR = 1.0).
       \note If this curve is left blank, then the curve is assumed to always equal 1.0.
```

Test object:
```
ElectricEquipment:ITE:AirCooled,
  Story 1 Core Space IT equipment 1,      !- Name
  Story 1 Core Thermal Zone,              !- Zone Name
  FlowFromSystem,                         !- Air Flow Calculation Method
  Watts/Area,                             !- Design Power Input Calculation Method
  ,                                       !- Watts per Unit {W}
  1,                                      !- Number of Units
  10,                                     !- Watts per Zone Floor Area {W/m2}
  ,                                       !- Design Power Input Schedule Name
  ,                                       !- CPU Loading  Schedule Name
  Curve Biquadratic 3,                    !- CPU Power Input Function of Loading and Air Temperature Curve Name
  0.4,                                    !- Design Fan Power Input Fraction
  0.0001,                                 !- Design Fan Air Flow Rate per Power Input {m3/s-W}
  Curve Biquadratic 4,                    !- Air Flow Function of Loading and Air Temperature Curve Name
  Curve Quadratic 2,                      !- Fan Power Input Function of Flow Curve Name
  22.5,                                   !- Design Entering Air Temperature {C}
  None,                                   !- Environmental Class
  AdjustedSupply,                         !- Air Inlet Connection Type
  ,                                       !- Air Inlet Room Air Model Node Name
  ,                                       !- Air Outlet Room Air Model Node Name
  Node 7,                                 !- Supply Air Node Name
  0,                                      !- Design Recirculation Fraction
  ,                                       !- Recirculation Function of Loading and Supply Temperature Curve Name
  1,                                      !- Design Electric Power Supply Efficiency
  ,                                       !- Electric Power Supply Efficiency Function of Part Load Ratio Curve Name
  1,                                      !- Fraction of Electric Power Supply Losses to Zone
  ITE-CPU,                                !- CPU End-Use Subcategory
  ITE-Fans,                               !- Fan End-Use Subcategory
  ITE-UPS,                                !- Electric Power Supply End-Use Subcategory
  5,                                      !- Supply Temperature Difference {deltaC}
  ,                                       !- Supply Temperature Difference Schedule
  2;                                      !- Return Temperature Difference {deltaC}
```

### Details
Some additional details for this issue (if relevant):
 - Platform (Operating system, version)
 - Version of EnergyPlus (if using an intermediate build, include SHA)
 - Unmethours link or helpdesk ticket number

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by mjwitte** [comment]
@jmarrec I think this is a duplicate of #7037 which was fixed in #7051.

**Comment by jmarrec** [comment]
Nice. Thanks for the link, I searched for "ElectricEquipmentITEAirCooled" and missed this one. tested that my file runs with current develop.
